site/content/en/reference/_index.md

```diff
@@ -93,6 +93,53 @@ By default, kpt will use the builtin schema.
   The path to an OpenAPI schema file. The default value is ./openapi.json
 ```
 
+### Annotations
+
+Kpt supports a small set of annotations that allow customizing the behavior
+of kpt on a per-resource basis
+
+#### config.kubernetes.io/local-config
```

Comment:
I suggest we hold off on this PR since the docs are giong through major changes. Specifically:

1. `function` annotation is going away.
2. `path` and `index` annotation are part of Functions Spec and the top-level reference page is the right place to discuss them.  They'll be discussed in Chapter 5 of the book.
3. We need to rationalize `local-config`. It seems a live-specific annotation, so it should probably discussed in the Chapter 6 of the book.

Reply:
https://docs.google.com/spreadsheets/d/1V5f1UpL0_WqJPyeFTaR0bR4acCGtoUENYHVvV_HOlrc/edit#gid=1380952439

Reply:
Seeing as the `function` annotation drives most of the useful Kpt functionality for writing complex Kpt function pipelines, is there any context available for people that _don't_ have access to that document?